ui_framework/doc_site/src/views/tool_bar/tool_bar.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import {
+  KuiToolBar,
+  KuiToolBarSearchBox,
+  KuiButton,
+  KuiButtonIcon,
+  KuiButtonGroup,
+} from '../../../../components';
+
+export const ToolBar = () => (
+  <KuiToolBar>
+    <KuiToolBarSearchBox onFilter={() => {}}/>
```

Comment:
FWIW, Watcher has a Select inside of a ToolBar. I'm OK with making this in a separate PR, but can write them down in a specific issue, so we don't forget?